apps/parks-app/src/bin/config.js

```
@@ -0,0 +1,19 @@
+
+  var db_config   = config.get('MONGODB_DB_URL'),
+      collection  = config.get('collection_name');
+
+  //normalize db connection string
+  if(db_config[db_config.length - 1] !== "/"){
+    db_config += '/';
+  }
+  config.add({db_config: db_config+collection});
+  return config;
+}
+exports = module.exports = autoconfig();
```

Biome (2.1.2)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)